plutus-core/plutus-ir/src/PlutusIR/Transform/Inline/CallSiteInline.hs

```diff
@@ -166,23 +129,23 @@ fullyApplyAndBetaReduce info args0 = do
         Term tyname name uni fun ann ->
         InlineM tyname name uni fun ann Bool
       safeToBetaReduce a = shouldUnconditionallyInline Strict a rhsBody
```

Comment:
This is an example of it being wrong. It's looking at the fully stripped body of the function, which isn't rigth.

Reply:
Do you think this https://github.com/input-output-hk/plutus/pull/5426/commits/ebb818ad067e712e59a0e5a174583f8ee9fae4a7 is more right? `acc` is the whole function that has been processed.

Reply:
yes, that looks more like it

Reply:
I continue to think that `varBody` should just be removed now.